app/fullService/api/getBalanceForAccount.ts

```diff
@@ -19,12 +19,22 @@ export function convertBalanceFromV2(accountStatus: AccountStatus): BalanceStatu
       Number(accountStatus.account.nextBlockIndex) >= Number(accountStatus.networkBlockHeight),
     localBlockHeight: accountStatus.localBlockHeight,
     networkBlockHeight: accountStatus.networkBlockHeight,
-    object: 'balance',
-    orphanedPmob: accountStatus.balancePerToken[TokenIds.MOB]?.orphaned || '0',
-    pendingPmob: accountStatus.balancePerToken[TokenIds.MOB]?.pending || '0',
-    secretedPmob: accountStatus.balancePerToken[TokenIds.MOB]?.secreted || '0',
-    spentPmob: accountStatus.balancePerToken[TokenIds.MOB]?.spent || '0',
-    unspentPmob: accountStatus.balancePerToken[TokenIds.MOB]?.unspent || '0',
+    balancePerToken: {
+      0: {
```

Comment:
there's no guarantee that FS will return a balance response for any given token. It only returns a key here for tokens for which this account has txos. For now it's okay to assume that only MOB and mUSD exist and manually fill those balance keys in if the account doesn't already have txos for a token